README.md

```diff
@@ -1,250 +1,103 @@
-```
 
-Each app reference in the `INSTALLED_APPS` setting must point to the
-corresponding ``AppConfig``. For example, instead of `'django.contrib.admin'`,
-the template uses `'<project_name>.apps.MongoAdminConfig'`.
-
-### Configuring migrations
-
-Because all models must use `ObjectIdAutoField`, each third-party and contrib app
-you use needs to have its own migrations specific to MongoDB.
-
-For example, `settings.py` in the project template specifies:
+This backend is currently in development and is not advised for Production workflows. Backwards incompatible
+changes may be made without notice. We welcome your feedback as we continue to
+explore and build. The best way to share this is via our [MongoDB Community Forum](https://www.mongodb.com/community/forums/tag/python).
 
-```python
-MIGRATION_MODULES = {
-    "admin": "mongo_migrations.admin",
-    "auth": "mongo_migrations.auth",
-    "contenttypes": "mongo_migrations.contenttypes",
-}
-```
+## Install
 
-The project template includes these migrations, but you can generate them if
-you're setting things up manually or if you need to create migrations for
-third-party apps. For example:
+Use the version of django-mongodb-backend that corresponds to your version of
+Django. For example, to get the latest compatible release for Django 5.0.x:
 
-```console
-$ python manage.py makemigrations admin auth contenttypes
-Migrations for 'admin':
-  mongo_migrations/admin/0001_initial.py
-    - Create model LogEntry
-...
-```
+`pip install --pre django-mongodb-backend==5.0.*`
 
-### Creating Django applications
+(Until the package is out of beta, you must use pip's `--pre` option.)
 
-Whenever you run `python manage.py startapp`, you must remove the line:
 
-`default_auto_field = 'django.db.models.BigAutoField'`
+## Quickstart
 
-from the new application's `apps.py` file (or change it to reference
- `"django_mongodb_backend.fields.ObjectIdAutoField"`).
+### Start Project
 
-Alternatively, you can use the following `startapp` template which includes
-this change:
+From your shell, run the following command to create a new Django project
+called example based on a custom template. Make sure the version referenced
+from `django-mongodb-labs` corresponds to your version of Django similar
+to the install stage. For example, this command works for Django 5.0.x:
 
 ```bash
-$ python manage.py startapp myapp --template https://github.com/mongodb-labs/django-mongodb-app/archive/refs/heads/5.0.x.zip
+$ django-admin startproject example --template https://github.com/mongodb-labs/django-mongodb-project/archive/refs/heads/5.0.x.zip
 ```
-(where "5.0" matches the version of Django that you're using.)
 
-### Configuring the `DATABASES` setting
 
-After you've set up a project, configure Django's `DATABASES` setting similar
-to this:
+### Connect to the database
+
+Navigate to your `example/settings.py` file and find the variable named `DATABASES` Replace the `DATABASES` setting with this:
```

Comment:
(one sentence you called DATABASES a variable, the next a setting.)
missing period after DATABASES.